pkg/kubectl/cmd/edit.go

```diff
@@ -131,7 +131,7 @@ func runEdit(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		return err
 	}
 
```

Comment:
Agree, was trying to make the refactor actually possible to follow, will add a TODO